src/Symfony/Bundle/FrameworkBundle/Translation/Translator.php

```diff
@@ -55,20 +54,8 @@ class Translator extends BaseTranslator implements WarmableInterface
      *
      * @throws InvalidArgumentException
      */
-    public function __construct(ContainerInterface $container, MessageSelector $selector, $defaultLocale = null, array $loaderIds = array(), array $options = array())
+    public function __construct(ContainerInterface $container, MessageSelector $selector, $defaultLocale, array $loaderIds = array(), array $options = array())
```

Comment:
it looks like the docblock is not up to date - and we could add a "string" type hint meanwhile here